src/it/streamingcommunity/src/eu/kanade/tachiyomi/animeextension/it/streamingcommunity/StreamingCommunity.kt

```diff
@@ -252,68 +256,89 @@ class StreamingCommunity : ConfigurableAnimeSource, AnimeHttpSource() {
 
     // ============================ Video Links =============================
 
-    override fun videoListRequest(episode: SEpisode): Request = GET("$baseUrl/watch/${episode.url}", headers)
+    object VideoRegex {
+        val PLAYLIST_URL_REGEX = Regex("""url: ?'(.*?)'""")
+        val EXPIRES_REGEX = Regex("""'expires': ?'(\d+)'""")
+        val TOKEN_REGEX = Regex("""'token': ?'([\w-]+)'""")
+        val TOKEN_QUALITY_REGEX = Regex("""'token(\d+p?)': ?'([\w-]+)'""")
+        val SUBTITLES_REGEX = Regex("""#EXT-X-MEDIA:TYPE=SUBTITLES.*?NAME="(.*?)".*?URI="(.*?)"""")
+    }
 
-    override fun videoListParse(response: Response): List<Video> {
-        val data = json.decodeFromString<VideoResponse>(response.asJsoup().getData())
+    @Deprecated("Use the non-RxJava API instead", replaceWith = ReplaceWith("getVideoList"))
+    override fun fetchVideoList(episode: SEpisode): Observable<List<Video>> {
```

Comment:
override getVideoList instead of fetchVideoList

Reply:
ready

Reply:
Not yet :)

Reply:
Now yes :)